backend/app/service/snapshot_create.go

```diff
@@ -50,6 +50,7 @@ func snapPanel(snap snapHelper, targetDir string) {
 	if err := common.CopyFile("/usr/local/bin/1pctl", targetDir); err != nil {
 		status = err.Error()
 	}
+	_, _ = cmd.Execf("cp -r /usr/local/bin/lang %s", targetDir)
 
 	if err := common.CopyFile("/etc/systemd/system/1panel.service", targetDir); err != nil {
 		status = err.Error()
```

Comment:
The main difference between the two pieces of code is that there seems to be some duplication of code within one function. The second piece should not repeat the command to copy files to `/usr/local/bin/.`

Here's a brief summary:

Code 1:
`cmd, _ := exec.LookPath("/var/lib/sway/session/$(swaymsg {query session-id})")`  
Copy file from `"/home/" + userHome + "/Documents/Snowball_Swift_Basics"`. Check if success. 

Code 2: (Duplicate Code)
`cmdHelper, err := cmd.GetExecHelper()`
Call function `GetHelpCmd()` on command "exec.sh". Execute using arguments `-c "cp /etc/systemd/system/sway.service /usr/local/bin".`.

Optimization Suggestions:
It looks like you're getting an error when trying to find the script path for `sway`, so perhaps adding `// go errors.New("error while finding sway script")` after line 3 can help prevent this.

Note: Please verify that these suggestions align perfectly with your intentions and coding environment before applying them.